src/code/z_actor.c

```diff
@@ -891,16 +891,25 @@ void func_8002D9F8(Actor* actor, SkelAnime* skelAnime) {
     actor->world.pos.z += sp1C.z * actor->scale.z;
 }
 
-s16 Actor_WorldYawTowardActor(Actor* actorA, Actor* actorB) {
-    return Math_Vec3f_Yaw(&actorA->world.pos, &actorB->world.pos);
+/**
+ * @return Yaw towards `actorTarget` for `actorOrigin`, using world positions.
+ */
+s16 Actor_WorldYawTowardActor(Actor* actorOrigin, Actor* actorTarget) {
```

Comment:
```suggestion
/**
 * @return Yaw from +z axis towards worldPos of `target` with the origin set as the worldPos of `origin`
 */
s16 Actor_WorldYawTowardActor(Actor* origin, Actor* target) {
```

Might need some tweaking, but I think it's worth getting into a bit more detail? The important bits are:
- the worldPos of `origin` acts as the origin the yaw is measured from.
- the yaw is measured from the +z axis towards the worldPos of `target` around the `origin`
- World Positions are used

Thoughts?